test/integration/targets/playbook/timeout.yml

```diff
@@ -0,0 +1,11 @@
+- hosts: localhost
+  gather_facts: false
+  tasks:
+    - shell: sleep 100
+      timeout: 1
+      ignore_errors: true
+      register: time
+
+    - assert:
+        that:
+            - time is failed
```

Comment:
Maybe check the message too, just so we have coverage that the right exception gets thrown.

Reply:
planning to add more, this was the 'most basic' i could think of